build.gradle

```diff
@@ -281,7 +281,7 @@ dependencies {
 
 
     antlr "org.antlr:antlr4:4.8"
-    spotbugs 'com.github.spotbugs:spotbugs:4.2.2'
+    spotbugs 'com.github.spotbugs:spotbugs:4.2.3'
```

Comment:
This is not the latest version of spotbugs: according to their [changelog](https://github.com/spotbugs/spotbugs/blob/master/CHANGELOG.md), that's 4.7.0.

But even bumping to 4.3.0 yields 955 violations, bumping to 4.7.0 yields 3k.

What is our approach to updating spotbugs?  I can ticket this separately.

Reply:
https://broadworkbench.atlassian.net/browse/DR-2644

Reply:
I think a change to spotbugs causes generated code to be scanned. If that's the case, there are probably a lot fewer fixes needed once the generated code is excluded from the scan.

Reply:
Nice, thanks!  Hope that's the case.